core/commons/che-core-commons-mail/src/main/java/org/eclipse/che/mail/EmailBean.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) 2012-2017 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.mail;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Describing e-mail properties.
+ *
+ * @author Igor Vinokur
+ * @author Alexander Garagatyi
+ */
+public class EmailBean {
+  private String from;
+  private String to;
+  private String replyTo;
```

Comment:
Do you know whether it supports multiple recipients?

Reply:
I don't